spec/connectors/registry_spec.rb

```diff
@@ -9,21 +9,79 @@
 require 'connectors/registry'
 
 describe Connectors::Factory do
-  let(:factory) do
-    described_class.new
+  subject { described_class.new }
+
+  let(:configuration) {
+    {}
+  }
+
+  let(:job_description) {
+    {}
+  }
+
+  let(:registered_connector) {
+    'my-connector'
+  }
+
+  let(:unregistered_connector) {
+    'another-connector'
+  }
+
+  class MyConnector
+    def initialize(configuration: {}, job_description: {})
+      # nothing needed
+    end
```

Comment:
```suggestion
    def initialize(configuration: {}, job_description: {}); end
```

This can be further simplified with empty method syntax - up to you to decide whether you want to use it here or not